### PR TITLE
Added multi-arch cluster-samples jobs to 4.7 periodics file

### PR DESCRIPTION
**Pull request by ravanelli**
Signed-off-by: Renata Ravanelli <rravanel@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 1]
```diff
@@ -7416,3 +7416,539 @@ periodics:
...
+            name: cluster-secrets-metal
+    - configMap:
+        name: prow-job-cluster-launch-installer-remote-libvirt-e2e
+      name: job-definition
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: release-pull-secret
+      secret:
+        secretName: ci-pull-credentials
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7554]: The s390x configs are being covered by the Z PRs :)

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 2]
```diff
@@ -7416,3 +7416,539 @@ periodics:
...
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7822]: Ditto about s390x.

File: core-services/template-deprecation/_allowlist.yaml [hunk 3]
```diff
@@ -1942,6 +1945,9 @@ templates:
           release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.6:
             generated: false
             kind: release
+          release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.7:
+            generated: false
+            kind: release
```
Comment (jaypoulz) [inline core-services/template-deprecation/_allowlist.yaml:1950]: s390x

File: core-services/template-deprecation/_allowlist.yaml [hunk 4]
```diff
@@ -1960,6 +1969,9 @@ templates:
           release-openshift-origin-installer-e2e-remote-libvirt-jenkins-e2e-s390x-4.6:
             generated: false
             kind: release
+          release-openshift-origin-installer-e2e-remote-libvirt-jenkins-e2e-s390x-4.7:
+            generated: false
+            kind: release
```
Comment (jaypoulz) [inline core-services/template-deprecation/_allowlist.yaml:1974]: s390x

**Review by jaypoulz — COMMENTED** [review]
Looks good for the power configs. Normally I'd be very happy to see the s390x configs added at the same time, but we're trying to keep the arches disjoint for now since we want our arch-specific PEs to "own" their test configs. :)

**Comment by ravanelli** [comment]
Done, removed all Z stuff, the PR is just covering POWER now.

**Review by jaypoulz — APPROVED** [review]
Looks good! Thanks!

**Comment by jaypoulz** [comment]
@ravanelli do you mind rebasing this?

**Review by petr-muller — APPROVED** [review]

**Pull request merged**